### PR TITLE
Aliasing `commands` to double tab when input is empty

### PR DESCRIPTION
**Pull request by MrBurrBurr**
As [requested](https://www.reddit.com/r/Steam/comments/at6erv/asf_feedback_evaluation_you_can_help_us_too/egz8q4v/) in the reddit feedback thread

### CODE REVIEW

**Comment by MrBurrBurr** [comment]
It would be possible. Not sure if thats what user expects, tho. Gonna wait for @Aareksio to confirm.

**Comment by Botan626** [comment]
Option to turn auto execute on/off in UI configuration?

**Comment by MrBurrBurr** [comment]
> Option to turn auto execute on/off in UI configuration?

No.

**Comment by deluxghost** [comment]
no botan, no shell will auto exec after tab pressing

File: src/views/Commands.vue [hunk 1]
```diff
@@ -237,6 +238,10 @@
 					const splitCurrentParameter = splitCommand[splitCommand.length - 1].split(',');
 
 					this.command = [...splitCommand.slice(0, -1), [...splitCurrentParameter.slice(0, -1), this.suggestedParameterValue].join(',')].join(' ');
+				} else if (this.command === '') {
+					let tabPressTime = new Date();
```
Comment (Aareksio) [inline src/views/Commands.vue]:
```suggestion
					let tabPressTime = Date.now();
```

Comment (MrBurrBurr) [reply src/views/Commands.vue]: Thanks!

**Pull request merged**